### DISCUSSION TITLE
G7 handoff to a new sensor

### DISCUSSION THREAD

**Discussion by jgslade** [NightscoutFoundation/xDrip#2896] [Q&A]
I've just started my 3rd G7, and both times I've done the hand off to the new sensor it's been a bit clunky.  Here is my procedure, if there's anything I can be doing better please let me know, thanks.

1. I'll insert the new sensor about the time the old one is set to expire then use the 12 hour grace period as a pre-soak time.
2. At the end of the 12 hour grace period, or like today an opportune time to switch, I'll stop the old sensor, wait 5 minutes then connect to the new one.
3. While it's trying to connect on the system status page says to stay on this page, after a couple of minutes I'll get the pairing request, I accept it. 
4. Then the system status page still says to stay on this page so I do until the stay on this page goes away when it starts getting readings, it's been in the 5-10 range so far.

Do I still need to stay on the page after I get the pairing request? It's kinda annoying to have to sit there with the phone screen on for 10 mins not doing anything.

I do like being without readings for only 15 minutes or so instead of 2+ hours with the g6, but if there's an easier way to do this I would love to know.  

Thanks.

**Comment by Navid200** [comment]
I believe after you approve the pair request, you can leave the status page.

**Reply by jgslade** [reply]
Cool, I'll give that a try next time. It'll be nice to turn the screen off after pairing.
